src/ducks/appsSuggestions/AppsSuggestion.js

```diff
@@ -0,0 +1,45 @@
+import { Document } from 'cozy-doctypes'
+import { head } from 'lodash'
+import { TRANSACTION_DOCTYPE } from 'doctypes'
+
+class AppsSuggestion extends Document {
+  static fetchBySlug(slug) {
+    return this.queryAll({ slug }).then(head)
+  }
+
+  static linkTransaction(suggestion, transaction) {
+    return {
+      ...suggestion,
+      relationships: {
+        ...suggestion.relationships,
+        transactions: {
+          data: [
+            ...suggestion.relationships.transactions.data,
```

Comment:
I think we should have a little helper that does the nested push.

`push('relationships.transactions.data', { _id: transaction._id, _type: TRANSACTION_DOCTYPE })`

Here I find it easy to erase data that we are not interested in. For example if `transactions` has other attributes than data, they will be erased.

```
const pushAtPath = (obj, path, value) => {
  const arr = get(obj, path) || []
  arr.push(value)
  set(obj, path, arr)
}
```